tide_core.install

```diff
@@ -372,3 +372,21 @@ function tide_core_update_8014() {
     $module_installer->install(['link_field_autocomplete_filter']);
   }
 }
+
+/**
+ * Enable hotlinking by default on Stage File Proxy.
+ */
+function tide_core_update_8015() {
+
+  $stage_file_proxy_config = \Drupal::config('stage_file_proxy.settings')->get();
```

Comment:
@FleetAdmiralButter It's worth wrapping this in a check to see if the module is active.
```suggestion
  if (\Drupal::moduleHandler()->moduleExists('stage_file_proxy')) {
```